library.rb

```diff
@@ -0,0 +1,150 @@
+require_relative 'rating'
+require_relative 'authors'
+require_relative 'book'
+require_relative 'clients'
+require_relative 'orders'
+require_relative 'file_handler'
+require_relative 'language'
+require 'date'
+require 'time'
+
+class Library < Orders
+  include Language
+  attr_reader :authors_lib, :books_lib, :clients_lib, :orders_lib
+
+  def initialize(authors_lib = Authors.new,
```

Comment:
It's better to have already preloaded arrays of classes.

Reply:
Done